main.py

```diff
@@ -0,0 +1,11 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+
+app = FastAPI()
+
+
+@app.get("/")
```

Comment:
Can you create some more endpoints and demonstrate the following:

1. What does a route that retrieves a file look like? Can you test this and create a simple file with lorem ipsum text in it?
2. Can you do a load test for routes that are dynamic in nature?  A route with randomly generated text as the parameter and the route just echos the input.
3. Can you demonstrate a route that would theoretically call a database?  Create a route that would simulate a call to a database that actually just is an arbitrary amount of time for retrieving data from a database between 300ms and 1 second (1000 ms)? You can accomplish this in most programming languages by doing something like Thread.sleep(timeToSleep)